pallets/spacewalk/Cargo.toml

```diff
@@ -41,6 +41,25 @@ orml-utilities = {default-features = false, git = "https://github.com/open-web3-
 pallet-transaction-payment = {git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.17"}
 substrate-stellar-sdk = {git = "https://github.com/pendulum-chain/substrate-stellar-sdk", branch = "polkadot-v0.9.17", default-features = false, features = ['offchain']}
 
+# for mock runtime
```

Comment:
Are all of these pallets required for mocking a simple runtime?

Reply:
Yeah I needed to include them just to get the thing to compile. I based it off the testchain. I created tickets to go through and minimize both the testchain and the mock runtime and remove as many features as possible. I'm sure it could be simplified it to be at least 10x smaller and remove most dependencies. Especially since spacewalk is just one pallet, not all of pendulum.